ts-tests/scripts/start-relay-and-para-chain.sh

```diff
@@ -19,11 +22,19 @@ $LITENTRY_BIN export-genesis-wasm > para-$PARACHAIN_ID-wasm
 
 # run alice and bob as relay nodes
 $POLKADOT_BIN --chain $ROCOCO_CHAINSPEC --alice --tmp --port 30333 --ws-port 9944 &> "$TMP_DIR/relay.alice.log" &
+echo $! > $RELAY_ALICE_PIDFILE
 sleep 3
+
 $POLKADOT_BIN --chain $ROCOCO_CHAINSPEC --bob --tmp --port 30334 --ws-port 9945  &> "$TMP_DIR/relay.bob.log" &
+echo $! > $RELAY_BOB_PIDFILE
 sleep 3
 
-# run a second litentry-collator instance
+# # run a second litentry-collator instance
 $LITENTRY_BIN --collator --tmp --parachain-id $PARACHAIN_ID --port 40333 --ws-port 9954 --alice -- --execution native --chain $ROCOCO_CHAINSPEC --port 30344 --ws-port 9946 &> "$TMP_DIR/para.alice.log" &
+echo $! > $PARA_ALICE_PIDFILE
 sleep 3
+
 $LITENTRY_BIN --collator --tmp --parachain-id $PARACHAIN_ID --port 40334 --ws-port 9955 --bob -- --execution native --chain $ROCOCO_CHAINSPEC --port 30344 --ws-port 9947 &> "$TMP_DIR/para.bob.log" &
+echo $! > $PARA_BOB_PIDFILE
```

Comment:
this might need to be rebased if #84 gets merged (i.e. we don't need bob parachain anymore)

Reply:
Yeah. I'd like to try `merge` this time

Reply:
#84 is merged now